srcts/build/_build.ts

```diff
@@ -79,7 +75,7 @@ async function build(
   return esbuildBuild({
     incremental: incremental,
     watch: watch,
-    target: "es5",
+    target: "es2020",
```

Comment:
Notes:
* [esbuild's `target`](https://esbuild.github.io/api/#target) does not polyfill the JavaScript input files. It only adjust _its_ javascript code. 
	* > Automatic polyfill injection is outside of esbuild's scope.
* Matches [tsconfig.json's `target`](https://github.com/rstudio/shiny/blob/d764ea9b4e4f773d7dc5c3bfc2e2741441b10b44/tsconfig.json#L4C1-L5C1) of `es2020`

Given typescript is converting the code to es2020 and esbuild is compiling to es2020, we should be safe!